src/main/java/seedu/linkedout/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,56 @@
+package seedu.linkedout.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.linkedout.logic.parser.CliSyntax.PREFIX_FIELD;
+import static seedu.linkedout.logic.parser.CliSyntax.PREFIX_ORDER;
+
+import java.util.Comparator;
+
+import seedu.linkedout.logic.commands.exceptions.CommandException;
+import seedu.linkedout.model.Model;
+import seedu.linkedout.model.applicant.Applicant;
+
+/**
+ * Sort the list of applciants in the linkedout app.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sort applicants in the LinkedOUT app. "
+            + "Parameters: "
+            + "[" + PREFIX_FIELD + "FIELD]"
+            + PREFIX_ORDER + "ORDER...\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_FIELD + "NAME "
+            + PREFIX_ORDER + "asc ";
```

Comment:
is this suppose to be capital letter?
```suggestion
            + PREFIX_ORDER + "ASC ";
```